views/members.handlebars

```diff
@@ -59,8 +49,10 @@
     </div>    
   </div>
 </div>
+<div id="bottom-spacer"></div>
```

Comment:
Just want to give a little room at the bottom of the page with this div so that the body-box doesn't go all the way to the bottom of the page. it looked very odd that way.